types/office-js-preview/index.d.ts

```diff
@@ -18588,6 +18588,55 @@ declare namespace Excel {
          */
         getRangeOrNullObject(ctx: Excel.RequestContext): Excel.Range;
     }
+    /**
+     *
+     * Provides information about the worksheet format change event.
+     *
+     * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+     * @beta
+     */
+    interface WorksheetRowHiddenChangedEventArgs {
+        /**
+         *
+         * Gets the range address that represents the changed area of a specific worksheet.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+        address: string;
+        /**
+         *
+         * Gets the change type that represents how the Changed event is triggered. See Excel.DataChangeType for details.
```

Comment:
Seems like this should be:
```suggestion
         * Gets the change type that represents how the Changed event is triggered. See Excel.RowHiddenChangeType for details.
```